index.html

```diff
@@ -64,6 +64,203 @@ <h3 class="supporting-text flex centered-content">
         </div>
         <div class="illustration-icon"></div>
       </section>
+      <!-- -work section -->
+      <section class="work-section">
+        <div class="subtitle flex">
+          <h2 class="subtitle">My Recent Works</h2>
+        </div>
```

Comment:
- [ ] Kindly the line must be under the title. fix that and let's make your project get approve 
Your design
![Screenshot from 2022-07-03 01-01-05](https://user-images.githubusercontent.com/77842237/177017312-7393508e-0f04-48a5-9c3c-732c182123ea.png)

Figma template 
![Screenshot from 2022-07-03 01-02-43](https://user-images.githubusercontent.com/77842237/177017324-c8226207-97f2-4461-9df7-4e5ef8d92c8a.png)